requirements.txt

```diff
@@ -1,94 +1,94 @@
 # dev requirements
 
 # sha256: SYQkQdJNiNDs_DhobQIcs7G-56v8VB6fxzReRiDe6rY
-Mako==0.7.3
+Mako==1.0.0
 # sha256: M57ATSCtnNzL_p843OYYLMUEznieXQ-WR-qnUvD5UwA
-MarkupSafe==0.15
+MarkupSafe==0.23
 # sha256: EWRYQrqOyYaujPvkxsrP9cNfD0Unq_T1WBrotK1JwLY
 Paste==1.7.5.1
 # sha256: vIdnT1rJli4O_pbwYLoioH4sYfvr9THvw4O4LbWykew
-SQLAlchemy==0.9.4
+SQLAlchemy==0.9.8
 # sha256: Wgo2x0fCj1A2asBG2kP1AFXITR2djtPAm5tbmmxRaDI
-alembic==0.6.4
+alembic==0.7.4
 # sha256: 3a9LCmGDNaMrZmTUrgOKHej7raOyUDP5AhUQ7Ss5QaQ
-argparse==1.2.1
+argparse==1.3.0
 
 # mock has both a zip and a tarball available, and which one we get is random.
 # sha256: lvpN706AIAvoJ8P1EUfdez-ohzuSB-MyXUe6Rb8ppcE
 # sha256: 6QTt-5DahBKcBiUs06BfkLTuvBu1uF7pblb_bPaUONU
-mock==0.8.0
+mock==1.0.1
 
 # sha256: TmPMMyXedc-Y_61AvnL6aXU96CRpUXMXj3TANP5PUmA
-nose==1.3.0
+nose==1.3.4
 # sha256: r1v0XaSpFtsrY4z_2enWZoszAg4rjKn4ZNt5tJMxxv8
 poster==0.8.1
 # sha256: nPgCbuv2CxrIy3j0q_87DfYNM3Co2ZHCrArz4F8lRgg
-sqlalchemy-citext==1.0-2
+sqlalchemy-citext==1.2-0
 # sha256: hHK_EwLbFb3yxDk6XpGAzb8ps_jtWwbAa_XGuo9DNYg
-mercurial==2.7.1
+mercurial==3.2.4
 # sha256: Yj_jJZOSGfYbQathJMV_SWDOT0hlFoRqqvfUd10qLto
 honcho==0.5.0
 # sha256: th84eGr2uNZEtTBVIBLWacdBX89YUt4yDykIANuXmD0
-gunicorn==18.0
+gunicorn==19.1.1
 
 # prod requirements
 # sha256: G4jdtVszwrmhMAnW9W-j8JcjtAqgxIF2D_jfh1tqA2s
 # sha256: HvSGayima8gi3CSoXzrGfGjaeUhPAsZqCyBSNMKEQ-4
 configman==1.2.8
 # sha256: UV_5I0YlkugyHfi0jEfjQo-NQG7iK43ne--WnRrxEXE
-configobj==4.7.2
+configobj==5.0.6
 # sha256: oz42dZy6Gowxw8AelDtO4gRgTW_xPdooH484k7I5EOY
 hbase-thrift==0.20.4
 # sha256: YjGaw0shgNRQpNknhzyyYA88E7zmsgd83XcMd7Oph-E
-isodate==0.4.7
+isodate==0.5.1
 # sha256: un_il03Cdt0bg-HEB9ZsetPuZDOPdT1PZXr8qWxJumw
-lxml==2.3.4
+lxml==3.4.1
 # sha256: jjAivJYKpdX8fUxWR4YIdpOhqZ8V4CWtkg_AqjeOHWc
 pgxnclient==1.2.1
 # sha256: Zm_VIvShBZ1bi0mW5Xb6eGL1BVKOEc8e9VRjEcYMPk0
-pika==0.9.8
+pika==0.9.14
 # sha256: SLcCp8pHnhvCwae4GHWgfUdCmBMmBZk-LLl59eCCd9c
-psycopg2==2.4.5
+psycopg2==2.5.4
 # sha256: U6gisBsElN-WWMBOhyFhgOXsvoWxeGc92Eix-AnVwQA
 # sha256: u5PYXKflDWjkGXhSu226YJi7dyUA-C9dsv9KgHEVBck
-elasticsearch==1.2
+elasticsearch==1.3.0
 # sha256: atb_lGMjAnslXevLZuPgNeVF1fhCe7Vji2LjskbU2Io
 # sha256: jEZclknV3j4XtGOFcipJBRERC2A8o8E62cdxMqRnxCw
 elasticsearch-dsl==0.0.2
 # sha256: 2upIE6eJSfnxlBvixW1aviyZqFgjUMrwWYBPtleIrKU
-pyelasticsearch==0.6.1
+pyelasticsearch==0.7.1
 # sha256: QQvqlt8kefozmGxAw-76nB57qVp_cHEcMVhgu-LWbH8
 # sha256: 2Fg3nvWYjUU0u4kJQy1pdCIQCq_ycimdZhM5g2ttrps
-urllib3==1.9.1
+urllib3==1.10
 # sha256: cHay6ATFKumO3svU3B-8qBMYb-f1_dYlR4OgClWntEI
-elasticutils==0.7
+elasticutils==0.10.2
```

Comment:
Please revert this, ElasticUtils > 0.7 breaks compatibility with our. And it is deprecated and will be removed this quarter, so no need to upgrade it. Also, it is possible that updating pyelasticsearch will cause problems as well, and as it is also deprecated, you might want to revert it too.